src/Tracy/assets/BlueScreen/bluescreen.phtml

```diff
@@ -49,9 +49,7 @@ $code = $exception->getCode() ? ' #' . $exception->getCode() : '';
 			<?php if ($exception->getMessage()): ?><p><?= Helpers::escapeHtml($title . $code) ?></p><?php endif ?>
 
 
-			<h1><?= Helpers::escapeHtml($exception->getMessage() ?: $title . $code) ?>
-			<a href="https://www.google.com/search?sourceid=tracy&amp;q=<?= urlencode($title . ' ' . preg_replace('#\'.*\'|".*"#Us', '', $exception->getMessage())) ?>" target="_blank" rel="noreferrer noopener">search&#x25ba;</a>
-			<?php if ($skipError): ?><a href="<?= Helpers::escapeHtml($skipError) ?>">skip error&#x25ba;</a><?php endif ?></h1>
+			<h1><?= trim(Helpers::escapeHtml($exception->getMessage() ?: $title . $code)) ?><a href="https://www.google.com/search?sourceid=tracy&amp;q=<?= urlencode($title . ' ' . preg_replace('#\'.*\'|".*"#Us', '', $exception->getMessage())) ?>" target="_blank" rel="noreferrer noopener">search&#x25ba;</a><?php if ($skipError): ?><a href="<?= Helpers::escapeHtml($skipError) ?>">skip error&#x25ba;</a><?php endif ?></h1>
```

Comment:
That's what I did originally but changed it to `white-space: pre` to handle not only line breaks but indentation as wel..